.github/workflows/nightly_tests.yaml

```diff
@@ -40,6 +40,8 @@ jobs:
     - uses: actions/setup-python@v5
       with:
         python-version: '3.10'
+    - name: Install expect package
```

Comment:
Should this package be mentioned in readme as dependency? Should it be added to makefile? If it is not crucial I would prefer not to add it

Reply:
It's only for testing, it shouldn't be added as dependency.